tests/vrf/test_vrf.py

```diff
@@ -325,6 +326,18 @@ def gen_vrf_fib_file(vrf, tbinfo, ptfhost, dst_intfs, \
 
     ptfhost.template(src="vrf/vrf_fib.j2", dest=render_file)
 
+def get_default_vrf_fib_dst_intfs(vrf, tbinfo):
+    dst_intfs = []
+    vms_num = len(tbinfo['topo']['properties']['topology']['VMs'])
+    if vrf == 'Vrf1':
+        dst_intfs_range = list(range(1, int(vms_num / 2) + 1))
```

Comment:
>2 [](http://example.com/codeflow?start=54&length=1)

Why "/2"? Could you add comment at function level?

Reply:
added a comment to function